engine/src/main/scala/cromwell/webservice/WebServiceUtils.scala

```diff
@@ -0,0 +1,61 @@
+package cromwell.webservice
+
+import akka.http.scaladsl.model.{Multipart, StatusCode}
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.marshalling.ToEntityMarshaller
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.server.Directives.complete
+import akka.stream.Materializer
+import akka.util.{ByteString, Timeout}
+import cromwell.webservice.routes.CromwellApiService._
+import cromwell.webservice.WorkflowJsonSupport.errorResponse
+import spray.json._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait WebServiceUtils {
```

Comment:
With a `trait` `WomtoolRouteSupport` can do
```
onComplete(materializeFormData(formData))
```
instead of
```
onComplete(WebServiceUtils.materializeFormData(formData))
``` 
which is shorter and makes it seem more like materializing form data is a fundamental capability of any class that's accepting forms (which I intend)